packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -54,7 +55,7 @@ export function getHostProps(element: Element, props: Object) {
     ...props,
     value: undefined,
     defaultValue: undefined,
-    children: '' + node._wrapperState.initialValue,
+    children: safeValueToString(getSafeValue(node._wrapperState.initialValue)),
```

Comment:
Can we just do `safeValueToString(node._wrapperState.initialValue` instead here?

Reply:
That seems correct. `_wrapperState` is only set once and that is already a `SafeValue`.